pkg/evaluator/builtin.go

```diff
@@ -74,6 +75,25 @@ func lenFunc(args []Value) Value {
 	return newError("'len' takes 1 argument of type 'string', array '[]' or map '{}' not " + args[0].Type().String())
 }
 
+var hasDecl = &parser.FuncDecl{
+	Name: "has",
+	Params: []*parser.Var{
+		{Name: "map", T: parser.ANY_TYPE},
+		{Name: "key", T: parser.STRING_TYPE},
+	},
+	ReturnType: parser.BOOL_TYPE,
+}
+
+func hasFunc(args []Value) Value {
+	if len(args) != 2 {
+		return newError("'has' takes 2 argument not " + strconv.Itoa(len(args)))
+	}
+	m := args[0].(*Map)
```

Comment:
Since the param is ANY, won't you need to do type validating here instead of assuming a `(*Map)` type assertion will work?

Reply:
Fixed.